### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by joshuacox**
quickstart for [issue1](https://github.com/Zayon/kanboard-redmine/issues/1), I have a Makefile in there that automates a few things using the cidfile:

```
make build   #build locally
make run   #run locally, you will be prompted for a local port
make logs  #get the logs from the running container
make enter #enter the running container
make clean #kill and remove the running container when you are through testing
```

### CODE REVIEW

**Comment by Zayon** [comment]
Thanks,

I will test this WE, maybe tonight.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	done ;
+
+TAG:
+	@while [ -z "$$TAG" ]; do \
+		read -r -p "Enter the tag you wish to associate with this container [TAG]: " TAG; echo "$$TAG">>TAG; cat TAG; \
+	done ;
+
+PORT:
+	@while [ -z "$$PORT" ]; do \
+		read -r -p "Enter the tag you wish to associate with this container [PORT]: " PORT; echo "$$PORT">>PORT; cat PORT; \
```
Comment (Zayon) [inline Makefile:64]: The string should be "Enter the **port** you wish to associate with this container [PORT]: "

**Comment by Zayon** [comment]
Hi,

It seems nginx doesn't have lecture rights on the plugin directory

`make logs`

> 2017/01/21 14:22:14 [error] 13#0: *1 FastCGI sent in stderr: "PHP message: PHP Warning:  include(/var/www/app/app/../plugins/Redmine/Template/config/integration.php): failed to open stream: Permission denied in /var/www/app/app/Core/Template.php on line 83
 
`make enter`

> bash-4.3# ls -la /var/www/app/plugins/
total 16
drwxr-xr-x    3 nginx    nginx         4096 Jan 21 14:21 .
drwxr-xr-x   13 root     root          4096 Jan 21 13:59 ..
-rw-r--r--    1 nginx    nginx           15 Jan 20 20:59 .gitignore
drwxr-xr-x    4 root     root          4096 Jan 21 13:59 Redmine

It would be nice if files "TAG", "NAME", "PORT" and "cid" were in a "docker" folder for example

Great work otherwise. :+1:

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM kanboard/kanboard
+
+ADD . /var/www/app/plugins/Redmine
+
```
Comment (Zayon) [inline Dockerfile:4]: RUN chmod 744 /var/www/app/plugins/Redmine

**Comment by joshuacox** [comment]
I added the chmod in, and just gitignored the NAME and TAG file as well for now (they are merely placeholders for the Makefile which keeps it generic so I can use it for lots of different projects)

**Comment by Zayon** [comment]
Great, you may have to update the pull request as the new commits are not included in this one.

**Comment by joshuacox** [comment]
there we go, I should not have named it 'Dockerfile' it made some ambiguous git comands and caused that failure, all good now

**Pull request merged**